be/src/exec/schema_scanner/schema_task_runs_scanner.cpp

```diff
@@ -135,29 +136,42 @@ Status SchemaTaskRunsScanner::fill_chunk(ChunkPtr* chunk) {
             break;
         }
         case 6: {
-            // DATABASE
+            // CATALOG
             {
                 ColumnPtr column = (*chunk)->get_column_by_slot_id(6);
+                std::string catalog_name = "default_catalog";
+                if (task_run_info.__isset.catalog) {
+                    catalog_name = task_run_info.catalog;
+                }
+                Slice value(catalog_name.c_str(), catalog_name.length());
+                fill_column_with_slot<TYPE_VARCHAR>(column.get(), (void*)&value);
+            }
+            break;
+        }
+        case 7: {
```

Comment:
should not change the order

Reply:
It will not affect compatibility, because the chunk is generated by `_s_tbls_columns` in this cpp file, not by FE.